### PR TITLE
Make country profile charts responsive

### PR DESCRIPTION
**Pull request by barshathakuri**
## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
         "@the-deep/deep-ui": "^2.0.0-beta.0",
```
Comment (tnagorra) [inline package.json:41]: Let's remove deep ui

File: app/views/GoodPractices/styles.css [hunk 2]
```diff
@@ -188,10 +188,12 @@
             .ordering-container {
                 display: flex;
                 justify-content: flex-end;
-
+
```
Comment (tnagorra) [inline app/views/GoodPractices/styles.css:191]: Please remove this whitespace

File: app/views/GoodPractices/index.tsx [hunk 3]
```diff
@@ -1057,27 +1040,86 @@ function GoodPractices(props: Props) {
...
+                                    emptyMessage={(
+                                        <div>
+                                            No Good Practice Found
+                                        </div>
+                                    )}
+                                    filteredEmptyMessage={(
+                                        <div>
+                                            No Good Practice Found
+                                        </div>
+                                    )}
```
Comment (tnagorra) [inline app/views/GoodPractices/index.tsx:1118]: the filtered empty message should be different than empty message

File: app/Base/index.tsx [hunk 4]
```diff
@@ -7,6 +7,7 @@ import { listToMap } from '@togglecorp/fujs';
 
 import 'mapbox-gl/dist/mapbox-gl.css';
 import '@the-deep/deep-ui/build/esm/index.css';
```
Comment (tnagorra) [inline app/Base/index.tsx:9]: Let's remove all dependencies from deep ui

File: app/components/SliderInput/styles.css [hunk 5]
```diff
@@ -2,6 +2,7 @@
     display: flex;
     position: relative;
     z-index: 0;
+    margin-top: var(--dui-spacing-medium);
```
Comment (tnagorra) [inline app/components/SliderInput/styles.css:5]: Why add a margin top?

File: app/views/CountryProfile/styles.css [hunk 6]
```diff
@@ -344,4 +348,14 @@
...
     }
+    @media (max-width: 1000px) {
+        .time-range-container {
+            padding-right: var(--dui-spacing-extra-large);
+        }
+        .select-input {
+            .input-section {
+                margin-right: var(--dui-spacing-extra-small);
+            }
+        }
```
Comment (tnagorra) [inline app/views/CountryProfile/styles.css:359]: Why add asymmetric margin and padding?

File: app/views/CountryProfile/index.tsx [hunk 7]
```diff
@@ -909,7 +909,7 @@ function CountryProfile(props: Props) {
                 </TooltipIcon>
             )}
             filters={(
-                <>
+                <div className={styles.timeRangeContainer}>
```
Comment (tnagorra) [inline app/views/CountryProfile/index.tsx:912]: Any reason to add a container here?

File: app/views/CountryProfile/index.tsx [hunk 8]
```diff
@@ -750,7 +750,7 @@ function CountryProfile(props: Props) {
                 </>
             )}
             filters={(
-                <>
+                <div className={styles.timeRangeContainer}>
```
Comment (tnagorra) [inline app/views/CountryProfile/index.tsx:753]: Any reason to add a container here?

**Review by tnagorra — CHANGES_REQUESTED** [review]
Please do not commit package-lock.json. We already use yarn.lock to track installed dependencies.

File: app/views/CountryProfile/index.tsx [hunk 9]
```diff
@@ -909,20 +909,17 @@ function CountryProfile(props: Props) {
...
-                        min={START_YEAR}
-                        labelDescription={`${conflictTimeRangeActual[0]} - ${conflictTimeRangeActual[1]}`}
-                        max={END_YEAR}
-                        step={1}
-                        minDistance={0}
-                        value={conflictTimeRangeActual}
-                        onChange={setConflictTimeRange}
-                    />
-                    <div />
-                    <div />
```
Comment (tnagorra) [inline app/views/CountryProfile/index.tsx:924]: We shouldn't remove this

File: app/views/GoodPractices/index.tsx [hunk 10]
```diff
@@ -474,7 +481,7 @@ function GoodPractices(props: Props) {
 
         return modifiedList;
     }, [goodPracticeResponse]);
-
+    console.warn('list', goodPracticeList);
```
Comment (tnagorra) [inline app/views/GoodPractices/index.tsx:484]: Let's remove the console

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/components/GoodPracticeItem/index.tsx [hunk 11]
```diff
@@ -55,6 +57,14 @@ function GoodPracticeItem(props: Props) {
...
                 alt=""
             />
+            {/* )}
+            {rendererClassName === 'list' && (
+                <img
+                    className={styles.previewList}
+                    src={image ?? gridCover}
+                    alt=""
+                />
+            )} */}
```
Comment (tnagorra) [inline app/components/GoodPracticeItem/index.tsx]: Let's just remove this block

File: app/views/GoodPractices/index.tsx [hunk 12]
```diff
@@ -1058,27 +1039,88 @@ function GoodPractices(props: Props) {
...
+                                    keySelector={goodPracticekeySelector}
+                                    rendererParams={goodPracticeRendererParams}
+                                    renderer={GoodPracticeItem}
+                                    errored={!!goodPracticeError}
+                                    pending={goodPracticeLoading}
+                                    messageShown
+                                    filtered={false}
+                                    emptyMessage={(
+                                        <div>
+                                            No Good Practice Found
```
Comment (tnagorra) [inline app/views/GoodPractices/index.tsx:1112]: No result found.

File: app/views/GoodPractices/index.tsx [hunk 13]
```diff
@@ -1058,27 +1039,88 @@ function GoodPractices(props: Props) {
...
+                                        </div>
+                                    )}
+                                    filteredEmptyMessage={(
+                                        <div>
+                                            No Filtered Good Practice Found
```
Comment (tnagorra) [inline app/views/GoodPractices/index.tsx:1117]: No result found for your filtering criteria.

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**